deployment/terraform/opensearch.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_opensearch_domain" "opensearch" {
+  domain_name    = "opensearch-domain"
+  engine_version = "OpenSearch_2.11"
+
+  cluster_config {
+    instance_type          = "t3.small.search"
+    instance_count         = 2
+    zone_awareness_enabled = true
+  }
+
+  ebs_options {
+    ebs_enabled = true
+    volume_size = 10
+  }
+
+  advanced_security_options {
+    enabled           = true
+    master_user_options {
+      master_user_arn = data.aws_elb_service_account.main.arn
```

Comment:
@protsack-stephan let me know if using Elastic Load Balancing account ARN is appropriate here.

Reply:
The intention was to not involve AWS Secrets Manager (at least for now) to pass sensitive data here. However, there might be more appropriate iam role here.

Reply:
From what I've read in the docs `arn` is optional, so I suggest we remove it.

Reply:
I've double checked that and got error while applying configs - https://github.com/opensupplyhub/open-supply-hub/actions/runs/9015585693/job/24770553111#step:6:256

It seems that removing `master_user_options ` with username and password or `master_user_arn` is not the case here.